utils/paramfetch/src/lib.rs

```diff
@@ -48,18 +48,25 @@ struct ParameterData {
 }
 
 #[inline]
-fn param_dir() -> String {
-    std::env::var(DIR_ENV).unwrap_or_else(|_| PARAM_DIR.to_owned())
+fn param_dir(data_dir: &Path) -> PathBuf {
```

Comment:
Let's add some helpful method documentation like for `get_params`. Maybe adding a verb to the method name would also improve its readability (`read_params_dir`?).